alembic/versions/2022100403_remove_is_superuser_from_user__22b6c41cd29c.py

```diff
@@ -0,0 +1,26 @@
+"""Remove is_superuser from User table
```

Comment:
This is not part of speeding up search bundles, right?

Reply:
Shoot. It is in a [different PR](https://github.com/codalab/codalab-worksheets/pull/4268) that I submitted Thursday (still hasn't been merged b/c needs review). I must have branched off from my local branch I used for that PR when I created the branch for this PR. Sorry about that! Perhaps I should revert those changes in this branch?

Reply:
Maybe you can get that PR in first, since it's pretty straightforward, and then you don't have to undo anything.

Reply:
OK, I'll ping the chat for a review. Thanks!